src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/spreadsheet/PricesSpreadsheetGenerator.kt

```diff
@@ -60,6 +61,10 @@ class PricesSpreadsheetGenerator(@Autowired private val template: JPCTemplatePro
                         .also { logger.info("Adding cancelled moves.") }
                         .apply { writeMoves(moves[5]) }
 
+            JourneysSheet(workbook, header)
+                    .also { logger.info("Adding journeys.") }
+                    .apply { writeJourneys(journeys) }
```

Comment:
Could inline call to get journeys I suppose so only hold in memory/scope as and when needed?